R-package/R/utils.R

```diff
@@ -311,17 +309,26 @@ lgb.check.obj <- function(params, obj) {
 
 }
 
+# [description]
+#     make sure that "metric" is populated on params,
+#     and add any eval values to itt
```

Comment:
`to itt` -> `to it`

In case you will create a PR to remove duplicated test, you can fix this typo at the same time 😃